pkg/validate/security_context.go

```diff
@@ -402,8 +424,43 @@ var _ = framework.KubeDescribe("Security Context", func() {
 
 			checkNetworkManagement(rc, containerID, false)
 		})
+	})
+
+	Context("SeccompProfilePath", func() {
+		var podID, profileDir, blockHostNameProfilePath, blockchmodProfilePath string
+		var podConfig *runtimeapi.PodSandboxConfig
+		var err error
+		sysCaps := []string{"ALL"}
+
+		BeforeEach(func() {
+			profileDir, err = createSeccompProfileDir()
+			if err != nil {
```

Comment:
Don't think we could just log and return, it will still run test then.
You should do assertion here `Expect(err).NotTo(HaveOccurred())`.

Reply:
@mikebrow This is the issue we need to fix.

Reply:
agreed will update ..